src/symbols.cpp

```diff
@@ -148,7 +148,7 @@ bool ElfParser::loadSymbolsUsingBuildId() {
     }
 
     ElfNote* note = (ElfNote*)at(section);
-    if (note->n_namesz != 4 || note->n_descsz < 2 || note->n_descsz > 64 || note->n_type != NT_GNU_BUILD_ID) {
+    if (note->n_namesz != 4 || note->n_descsz < 2 || note->n_descsz > 64) {
```

Comment:
When I tested various Linux distributions and gcc versions I've found that NT_GNU_BUILD_ID is not always defined in <elf.h>. Well, I could define it myself, but the check seems redundant, because the section here is known to be SHT_NOTE with the name ".note.gnu.build-id".